jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -369,7 +369,7 @@ private static boolean isSamePackage(Class<?> a, Class<?> b){
 		}
 		
 		void checkAccess(MethodHandle handle, boolean skipAccessCheckPara) throws IllegalAccessException {
-			if (INTERNAL_PRIVILEGED == accessMode) {
+			if (INTERNAL_PRIVILEGED == accessMode || FULL_ACCESS_MASK == accessMode) {
```

Comment:
Please add extra parentheses here and in similar tests on lines 391 amd 405.

Reply:
Fixed.